pkger/parser.go

```diff
@@ -162,7 +162,7 @@ func parseSource(r io.Reader, opts ...ValidateOptFn) (*Pkg, error) {
 		strings.Contains(contentType, "yml"):
 		return parseYAML(bytes.NewReader(b), opts...)
 	default:
-		return parseJsonnet(bytes.NewReader(b), opts...)
+		return parseYAML(bytes.NewReader(b), opts...)
```

Comment:
this may end up changing to something else in the future. But right now most folks are using yml, and if the content type is failed to detect or detected as `application/octet-stream` then feels best to fall back to yaml for the time being until we have metrics that tell us we have a different encoding that is more common and we can switch it to be that one if desired.